### DISCUSSION TITLE
I accidentally deleted the driver from device manager

### DISCUSSION THREAD

**Discussion by N0rthTower** [nefarius/ViGEmBus#145] [Support]
I deleted the device driver from device manager and now it shows as unknown device and does not work with any of my programs
![image_2022-08-04_202931914](https://user-images.githubusercontent.com/71353628/182936838-468c0b79-7b61-4805-958a-6a8361e0a515.png)

**Comment by nefarius** [comment]
Reinstall?

**Reply by N0rthTower** [reply]
I tried that and it didn't work.

**Reply by nefarius** [reply]
https://vigem.org/projects/ViGEm/How-to-Install/
